### PR TITLE
Feat/dev 828 create a my nf ts tab

### PR DESCRIPTION
**Pull request by Kimmyyoung**
#### 🖥️ Summary
Create MyNfts Tab - in the collection header

[JIRA-TICKET-DEV-828](https://rebustoken.atlassian.net/browse/DEV-828?atlOrigin=eyJpIjoiYTA2MzA1YzAzYmI1NDc5MTg3Mzk5NzYxMzYxNzE1N2QiLCJwIjoiaiJ9)

#### 📝 Preview

Screenshot 2024-05-15 at 8 39 12 AM

Screenshot 2024-05-15 at 8 39 29 AM

#### 🖌️ Design 

image

### CODE REVIEW

**Comment by Linetes** [comment]
Nice!

**Review by pierstab — DISMISSED** [review]

**Review by cmdnotfound — DISMISSED** [review]
LGTM

File: apps/web/src/views/Nft/market/Collection/MyNFTs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const Traits = dynamic(() => import('../Traits'), {
+  loading: () => <PageLoader />,
+})
+// const Activity = dynamic(() => import('./Activity'), {
+//   loading: () => <PageLoader />,
+// })
+
+const getHashFromRouter = (router: NextRouter) => router.asPath.match(/#([a-z0-9]+)/gi)
+
+const Collection = () => {
```
Comment (yuushazen) [inline apps/web/src/views/Nft/market/Collection/MyNFTs/index.tsx:19]: Is this being used?

Comment (Kimmyyoung) [reply apps/web/src/views/Nft/market/Collection/MyNFTs/index.tsx:19]: I just checked my side, and I don't think we are using this, I can confirm with Cesar! and will do update it if we don't need it

**Review by cmdnotfound — APPROVED** [review]
LGTM

**Review by pierstab — APPROVED** [review]

**Pull request merged**